codecov_auth/commands/owner/interactors/start_trial.py

```diff
@@ -6,20 +6,20 @@
 
 
 class StartTrialInteractor(BaseInteractor):
-    def validate(self, owner: Owner):
-        if not owner:
+    def validate(self, current_org: Owner):
+        if not current_org:
             raise ValidationError("Cannot find owner record in the database")
 
-    def _start_trial(self, owner: Owner) -> None:
-        plan_service = PlanService(current_org=owner)
-        plan_service.start_trial()
+    def _start_trial(self, current_org: Owner) -> None:
+        plan_service = PlanService(current_org=current_org)
+        plan_service.start_trial(current_owner=self.current_owner)
         return
 
     @sync_to_async
     def execute(self, org_username: str) -> None:
```

Comment:
Optional: adjust org_username to current_org_username, also at the interactor/command level